docs/github.md

```diff
@@ -33,6 +36,9 @@ within Rust to suppress the lint.
 If changes are made to the `Dockerfile` then the CI will automatically rebuild
 the Docker image and run your tests on it.
 
+## Local Development
```

Comment:
This should be before `CI Behaviors` and should point to the installation flow page as well. Something like: "Once you've set up Calyx for contributing <link>, install the git hooks.

If we're recommending `seutp_hooks.sh`, let's make sure it runs `cargo clippy` and `black` on the related files.

Reply:
<img width="919" alt="Screenshot 2024-11-06 at 10 17 58 AM" src="https://github.com/user-attachments/assets/5a15cdf0-94e1-49b3-89a9-843bbcaa5670">

I'll move it!